Update Marketo Static Lists and DV360 deprecation notices. (#6685)
segmentio/segment-docs: src/connections/destinations/catalog/marketo-static-lists/index.md

---

> warning "Deprecation Notice"

> > This destination has been deprecated. Segment has created an instance of the [Marketo Static Lists (Actions)](/docs/connections/destinations/catalog/actions-marketo-static-lists/) destination for each properly configured version of the Personas Marketo Static Lists classic destination in your workspace. Settings and configurations have been migrated automatically. If your Personas Marketo Static List destination is connected to a Journey, please change your configuration and connect it to the new Actions destination. 

Reviewer: ```suggestion
> This destination has been deprecated. Segment created an instance of the [Marketo Static Lists (Actions)](/docs/connections/destinations/catalog/actions-marketo-static-lists/) destination for each properly configured version of the Personas Marketo Static Lists classic destination in your workspace. Settings and configurations have been migrated automatically. If your Personas Marketo Static List destination is connected to a Journey, edit your Journey to reflect to the new Actions destination. 
```

> > This destination has been deprecated. Segment created an instance of the [Marketo Static Lists (Actions)](/docs/connections/destinations/catalog/actions-marketo-static-lists/) destination for each properly configured version of the Personas Marketo Static Lists classic destination in your workspace. Settings and configurations have been migrated automatically. If your Personas Marketo Static List destination is connected to a Journey, edit your Journey to replace the Personas destination with the new Actions destination. 

Reviewer: ```suggestion
> This destination has been deprecated. Segment created an instance of the [Marketo Static Lists (Actions)](/docs/connections/destinations/catalog/actions-marketo-static-lists/) destination for each properly configured version of the Personas Marketo Static Lists classic destination in your workspace. Settings and configurations have been migrated automatically. If your Personas Marketo Static List destination is connected to a Journey, [create a new version of your Journey](/docs/engage/journeys/build-journey/#working-with-a-published-journey) to replace the Personas destination with the new Actions destination. 
```

> 

> > We have disabled all existing Personas Marketo Static List destinations, except those connected to a Journey instance. You can still access your existing configuration, but you will not be able to make changes to it. You will also no longer be able to create new instances of Personas Marketo Static Lists. Please refer to [Marketo Static Lists (Actions)](/docs/connections/destinations/catalog/actions-marketo-static-lists/) to set up a new instance of Marketo Static Lists. 

Reviewer: ```suggestion
> Segment disabled all existing Personas Marketo Static List destinations, except those connected to a Journey instance. You can still access your existing configuration, but you will not be able to make changes to it. You will also no longer be able to create new instances of Personas Marketo Static Lists. Please refer to [Marketo Static Lists (Actions)](/docs/connections/destinations/catalog/actions-marketo-static-lists/) to set up a new instance of Marketo Static Lists. 
```
